arches/management/commands/packages.py

```diff
@@ -1114,6 +1124,128 @@ def import_reference_data(self, data_source, overwrite="ignore", stage="stage",
+                if value and is_fk and model == models.ControlledListItem: 
+                    if field_name == "controlled_list":    
+                        related_list = models.ControlledList.objects.get(id=value)
+                        setattr(instance, field_name, related_list)
+                    elif field_name == "parent":
+                        # stash list items with parent relationships to create relationships after all list items have been created
+                        # stashed object in the form of {child_list_item_instance : parent_list_item_pk, ...}
+                        list_items_with_parent[instance] = value
+                elif value and is_fk and model == models.ControlledListItemValue:
+                    if field_name == "valuetype":
+                        valuetype = models.DValueType.objects.get(valuetype = value)
+                        setattr(instance, field_name, valuetype)
+                    elif field_name == "language":
+                        related_language = models.Language.objects.get(code=value)
+                        setattr(instance, field_name, related_language)
+                    else:
+                        related_list_item = models.ControlledListItem.objects.get(id=value)
+                        setattr(instance, field_name, related_list_item)
+                else:
+                    setattr(instance, field_name, value)
+
+            # run validation on all non-parent fields & gather for bulk create
+            instance.clean_fields(exclude={"parent"})
+            instances.append(instance)
+        
+        model.objects.bulk_create(instances)
+
+        if model == models.ControlledListItem:
+            # Create list item relationships after all list items have been created
+            for child, parent in list_items_with_parent.items():
+                child.parent = models.ControlledListItem.objects.get(id=parent)
+                child.clean_fields(exclude={field["name"] for field in fields if field["name"] != "parent"})
+                child.save()
+        
+    
+    def export_controlled_lists(self, data_dest):
+        wb = openpyxl.Workbook()
+        ws = wb.active
+        ws.title = "ControlledList"
+        self.export_model_to_sheet(ws, models.ControlledList)
+        self.export_model_to_sheet(wb, models.ControlledListItem)
+        self.export_model_to_sheet(wb, models.ControlledListItemValue)
+        
+        # if data_dest == ".":
+        #     data_dest = os.path.dirname(settings.SYSTEM_SETTINGS_LOCAL_PATH)
+        if data_dest != "" and data_dest != ".":
+            wb.save(os.path.join(data_dest, "controlled_lists.xlsx"))
+            self.stdout.write("Data exported successfully to controlled_lists.xlsx")
+        else:
+            utils.print_message("No destination directory specified. Please rerun this command with the '-d' parameter populated.")
```

Comment:
Ideally we'd find a way to do this with `self.stdout.write()`, but if you want to keep the "*" * 80 I guess you could still factor that in.